SpatialGDK/Source/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -51,9 +51,34 @@ bool USpatialGameInstance::HasSpatialNetDriver() const
 	return bHasSpatialNetDriver;
 }
 
+void SpatialBrowse(FWorldContext& WorldContext, FURL URL)
+{
+	bool bIsSpatial = false;
+	UNetDriver* TempNetDriver = nullptr;
+	if (GEngine->CreateNamedNetDriver(WorldContext.PendingNetGame, NAME_PendingNetDriver, NAME_GameNetDriver))
+	{
+		TempNetDriver = GEngine->FindNamedNetDriver(WorldContext.PendingNetGame, NAME_PendingNetDriver);
+		// Setting the PendingNetGame's NetDriver is necessary here because the call to CreateNamedNetDriver above will interfere
+		// with the internals of InitNetDriver and cause the NetDriver not to be initialized, and fail a check().
+		WorldContext.PendingNetGame->NetDriver = TempNetDriver;
+		bIsSpatial = TempNetDriver->IsA(USpatialNetDriver::StaticClass());
```

Comment:
You should use `GetDefault<UGeneralProjectSettings>()->bSpatialNetworking;` instead.

Reply:
Done, cheers :D